builtins/amp-layout.js

```diff
@@ -15,18 +15,22 @@
  */
 
 import {BaseElement} from '../src/base-element';
-import {isLayoutSizeDefined} from '../src/layout';
+import {Layout, isLayoutSizeDefined} from '../src/layout';
 import {registerElement} from '../src/service/custom-element-registry';
 
 class AmpLayout extends BaseElement {
 
   /** @override */
   isLayoutSupported(layout) {
-    return isLayoutSizeDefined(layout);
+    return layout == Layout.CONTAINER ||
+        isLayoutSizeDefined(layout);
   }
 
   /** @override */
   buildCallback() {
+    if (this.getLayout() == Layout.CONTAINER) {
```

Comment:
needs validator changes in `validator-main.protoascii` and `amp-layout.md`

Reply:
done